server/routes/index.js

```diff
@@ -213,6 +213,31 @@ router.get('/api/v1/search/studies', ensureAuthenticated, function (req, res) {
   })
 })
 
+router.get('/api/v1/subjects', ensureAuthenticated, function (req, res) {
+  const { dataDb } = req.app.locals
+  dataDb
+    .collection('metadata')
+    .aggregate([
+      { $match: { study: { $in: JSON.parse(req.query.q) } } },
+      {
+        $addFields: {
+          numOfSubjects: { $size: { $ifNull: ['$subjects', []] } },
```

Comment:
This adds a param for number of subjects fetched? And if subjects is null it returns `$subjects`?

Reply:
This wasn't anything that we did, but it seems that way, that if the property subjects of this document is null then we return an empty array